contracts/LifMarketMaker.sol

```diff
@@ -162,7 +213,7 @@ contract LifMarketMaker is Ownable {
   }
 
   function() payable {
-    buyLif();
+    getTokens();
   }
 
   function getTokens() payable {
```

Comment:
In this function, `getBuyPrice` is being used when `getSellPrice` should be used instead. Also we have to add profit calculation here, but if you want to leave it out from this PR that's fine to me